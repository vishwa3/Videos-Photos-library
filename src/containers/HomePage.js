import React from 'react';
import { Link } from 'react-router-dom';

const HomePage = () => (
  <div className="jumbotron center">
    <h1 style={{fontFamily:'Pacifico'}} className="lead">Welcome to Media Library</h1>
    <div>
      <ul className="ul-home">
       <li><Link to="/searchImages"><i class="fas fa-image fa-2x"></i><span className="i-home">Check out our HD Images</span></Link></li>
       <li><Link to="/searchVideos"><i  class="fas fa-video fa-2x"></i><span className="i-home">Watch Videos</span></Link></li>
       <li><Link to="/about"><i className="i-home" class="fa fa-info-circle fa-2x" aria-hidden="true"></i><span className="i-home">About</span></Link></li>
      </ul>
    </div>
  </div>
);

export default HomePage;