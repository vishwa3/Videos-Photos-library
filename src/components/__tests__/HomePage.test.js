import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import HomePage from '../HomePage';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('HomePage', () => {
  test('snapshot for HomePage', ()=> {
    const component = renderer.create(<Router><HomePage /></Router>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})