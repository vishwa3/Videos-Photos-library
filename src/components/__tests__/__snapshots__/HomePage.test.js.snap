// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage snapshot for HomePage 1`] = `
<div
  className="jumbotron center"
>
  <h1
    className="lead"
    style={
      Object {
        "fontFamily": "Pacifico",
      }
    }
  >
    Welcome to Media Library built with React, Redux, and Redux-saga 
  </h1>
  <div>
    <a
      href="/library"
      onClick={[Function]}
    >
      <button
        className="btn btn-lg btn-primary"
      >
         Visit Library
      </button>
    </a>
  </div>
</div>
`;
